designer/client/i18n/translations/en.translation.json

```diff
@@ -62,6 +62,10 @@
       "helpText": "Tick this box if users do not need to complete this field to progress through the form",
       "title": "Make {{component}} optional"
     },
+    "componentOmitSummaryOption": {
+      "helpText": "Tick this box to omit this component from the summary page",
```

Comment:
I don't think this text exists anywhere on the current application. Might need @jenbutongit to take a second look at it.

Reply:
[designer/client/field-edit.tsx](https://github.com/XGovFormBuilder/digital-form-builder/pull/799/files#diff-a4067012fdbd149eacd42f2e888e9bad1d9380c300bf7e621e9f52242abc0fb7)

This is where its currently being used